packages/types/src/documents/global/config.ts

```diff
@@ -79,6 +79,8 @@ export interface OIDCConfigs {
   configs: OIDCInnerConfig[]
 }
 
+export interface OIDCLogosConfig extends Config {}
```

Comment:
Is there any fields we can add to here?

Reply:
Should add config specific fields to `OIDCLogosInnerConfig` and define this type as 
```ts
export interface OIDCLogosConfig extends Config<OIDCLogosInnerConfig> {}
```